app/facade/food_facade.rb

```diff
@@ -0,0 +1,9 @@
+class FoodFacade
+  
+  def self.food_search(food)
+    parsed = FoodService.search(food)
+    parsed[:foods].first(10).map do |data|
```

Comment:
Nice job limiting here but see my note in your Service about adding a param to limit your page size to 10 at the call.